README.md

```diff
@@ -27,9 +27,10 @@ make docker-nats
 The locally built Docker image can then be used in place of the published Docker image in your compose file.
 See [Compose Builder](https://github.com/edgexfoundry/edgex-compose/tree/main/compose-builder#gen) `nat-bus` option to generate compose file for NATS and local dev images.
 
-## REST Endpoints
+## REST Server
```

Comment:
Please revert this change. Original name is correct.

Reply:
Create two sections and this. Existing one is Async and new one is Commands. In Commands device service endpoint is `/api/v2/device/name/{deviceName}/{resourceName}`. Add text about GET and PUT to these endpoints and how the service translates those to GET and PUT to the end REST device.